MainWindow.xaml.cs

```diff
@@ -0,0 +1,2329 @@
+                    case "ff":
+                        wynik = "14";
+                        break;
+                }
+            }
+            if (kolumna == 6)
+            {
+                switch (litera)
+                {
+                    case "v":
+                        wynik = "1";
+                        break;
+                    case "h":
+                        wynik = "2";
+                        break;
+                    case "bb":
+                        wynik = "3";
+                        break;
+                    case "j":
+                        wynik = "4";
+                        break;
+                    case "n":
+                        wynik = "5";
+                        break;
+                    case "z":
+                        wynik = "6";
+                        break;
+                    case "dd":
+                        wynik = "7";
+                        break;
+                    case "ff":
+                        wynik = "8";
+                        break;
+                    case "o":
+                        wynik = "9";
+                        break;
+                }
+            }
+            if (kolumna == 8)
+            {
+                switch (litera)
+                {
+                    case "t":
+                        wynik = "1";
+                        break;
+                    case "f":
+                        wynik = "0";
+                        break;
+                }
+            }
+            if (kolumna == 9)
+            {
+                switch (litera)
+                {
+                    case "t":
+                        wynik = "1";
+                        break;
+                    case "f":
+                        wynik = "0";
+                        break;
+                }
+            }
+            if (kolumna == 11)
+            {
+                switch (litera)
+                {
+                    case "t":
+                        wynik = "1";
+                        break;
+                    case "f":
+                        wynik = "0";
+                        break;
+                }
+            }
+            if (kolumna == 12)
+            {
+                switch (litera)
+                {
+                    case "g":
+                        wynik = "1";
+                        break;
+                    case "p":
+                        wynik = "2";
+                        break;
+                    case "s":
+                        wynik = "3";
+                        break;
+                }
+            }
+            if (kolumna == 15)
+            {
+                switch (litera)
+                {
+                    case "+":
+                        wynik = "1";
+                        break;
+                    case "-":
+                        wynik = "0";
+                        break;
+                }
+            }
+
+            return wynik;
```

Comment:
moglbys to w configu trzymac i pozniej sprawdzac czy jest intem i na ktorej kolumnie stoi odszukujesz w configu i zamieniasz